modules/era/sql/era_mob_droplist.sql

```diff
@@ -11827,6 +11827,8 @@ INSERT INTO `mob_droplist` VALUES (1450,0,0,1000,1527,@ALWAYS);  -- Behemoth Ton
 INSERT INTO `mob_droplist` VALUES (1450,0,0,1000,831,@VCOMMON);  -- Square Of Shining Cloth (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (1450,0,0,1000,860,@ALWAYS);   -- Behemoth Hide (Always, 100%)
 INSERT INTO `mob_droplist` VALUES (1450,0,0,1000,860,@COMMON);   -- Behemoth Hide (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (1450,0,0,1000,883,@ALWAYS);   -- Behemoth Horn (Always, 100%)
```

Comment:
prev didnt drop at all?

Reply:
> prev didnt drop at all?

At least I did not find, I think items on live now are from the KSNM99.